include/cantera/oneD/Domain1D.h

```diff
@@ -196,8 +196,19 @@ class Domain1D
         m_name[n] = name;
     }
 
-    //! index of component with name `name`.
-    virtual size_t componentIndex(const string& name) const;
+    /**
+     * Index of component with name `name`.
+     * @param name  name of component
+     * @param checkAlias  if `True` (default), check alias mapping
+     */
+    virtual size_t componentIndex(const string& name, bool checkAlias=true) const;
```

Comment:
```suggestion
    /**
     * Index of component with name `name`.
     * @param name  name of component
     * @param checkAlias  if `true` (default), check alias mapping
     */
    virtual size_t componentIndex(const string& name, bool checkAlias=true) const;
```
Also, should add a `@since` annotation for the new argument. Ditto for `hasComponent` below.